pkg/storage/utils_test.go

```diff
@@ -0,0 +1,192 @@
+package storage
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetMachineReadableFormat(t *testing.T) {
+	tests := []struct {
+		name         string
+		storageName  string
+		storageSize  string
+		mountedPath  string
+		activeStatus bool
```

Comment:
I think `mountStatus` would make more sense, and it goes inline with the value of `name`.

Reply:
Removed

Reply:
I wasn't suggesting you remove it, I was just looking for a better variable/property name.